storage/file.go

```diff
@@ -55,6 +170,58 @@ func (f FileServiceClient) createShare(name string) (*storageResponse, error) {
 	return f.client.exec("PUT", uri, headers, nil)
 }
 
+// GetShareProperties provides various information about the specified
+// file. See https://msdn.microsoft.com/en-us/library/azure/dn689099.aspx
+func (f FileServiceClient) GetShareProperties(name string) (*ShareProperties, error) {
+	verb := "HEAD"
```

Comment:
@enieuw Can we have this directly in `.exec` instead of defining it here like we did in other places?

```Go
resp, err := f.client.exec("PUT", uri, headers, nil)
return f.client.exec("DELETE", uri, f.client.getStandardHeaders(), nil)
```